### ISSUE TITLE
C1-2738-0 / Crunchyroll

### ISSUE THREAD

**Issue by Mirlo** [TheOfficialFloW/NoNpDrm#23]
Only an issue with Crunchyroll on 3.65.
Crunchyroll launched without issue on my 3.60 vita tv.

Tried installing Crunchyroll w/out mod through pkgj and received this error:
wrong size of zRIF, is it corrupted?

All my other nonpdrm dumps work.

**Comment by TheOfficialFloW** [comment]
Sony removed support for this app iirc

**Comment by Mirlo** [comment]
Any other applications that may be unusable in 3.65?

**Comment by TheRadziu** [comment]
It said wrong size or zRIF corrupt because it has "NOT REQUIRED" as zRIF. pkgj doesnt really handle that well (will try to replace all NOT REQUIRED with actual zRIFs since nonpdrm still creates license for free titles, even if they were "refreshed" through vitashell without any license. 

I was talking with someone over discord about modded crunchyroll and it seems it does work just fine on 3.65 as long as its not modded at all (untouched /app/gameid and untouched /patch/gameid) and crashes with any mod[didnt use RePatch to remake the mod, so cant tell if it'll work or not just yet. 

tl;dr it crashed because non-pfs update mixed with modded files in /patch/ since it doesnt work on 3.65 anymore, if you reinstall clean CR it should work just fine and nonpdrm plugin is not the one to blame.
